Scala/Scala.sublime-syntax

```diff
@@ -7,6 +7,9 @@ file_extensions:
   - sbt
 scope: source.scala
 variables:
+  # lookahead/behind for operators
+  oplookbehind: '(?<=[[[:alpha:]]0-9\s\(\)\[\]\{\}])'
```

Comment:
That's going to be a problem.  `:` and `=>` could be valid following almost any token.  There really isn't a way for us to backtrack over a context (is there?).